package.json

```diff
@@ -11,7 +11,7 @@
   ],
   "main": "index.js",
   "scripts": {
-    "test": "node test.js | tspec"
+    "test": "grunt && node test.js | tspec"
```

Comment:
"test": "jscs index.js test.js examples/ && node test.js | tspec"

then no need for grunt (though jscs would now need to be a dev dep)